6.X/6.7.0/release_notes.md

```diff
@@ -0,0 +1,35 @@
+Release 6.7.0 Dec, 2023
+===
+
+Review the full feature [release notes](https://docs.sysdig.com/en/sysdig-on-premises-release-notes.html).
+
+## Using MinIO in the Sysdig On-premises Environments
+
+Starting from release v6.6.0 we have added MinIO to the Sysdig stack (specifically importing the MinIO binary from upstream) for use in conjunction with our services.
+
+Download the MinIO source code from [minio](https://github.com/minio/minio). It is licensed under the [AGPL 3.0](https://github.com/minio/minio/blob/master/LICENSE).
+
+Copyright: MinIO Project, (C) 2015-2023 MinIO, Inc. This product includes software developed at [MinIO, Inc](https://min.io/)
+
+### Sysdig Agent Version
+
+Qualified with agent release
```

Comment:
Please add the agent release version

Reply:
AFAIK the agent should be any supported version, we never specify any version here, agent and onprem are independent, correct me if I'm wrong @figarocorso

Reply:
Qualified with agent release - is a sentence fragment

Reply:
I am OK removing this. Let's internally discuss if we can improve onprem/agent compatibility docs (maybe we could just link other release notes or docs site so we keep this repo simpler). Thank you for being aware.